pkg/cmd/grafana-cli/commands/scuemata_cuetsy_command.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/utils"
+	"github.com/grafana/grafana/pkg/schema"
+	"github.com/grafana/grafana/pkg/schema/load"
+	"github.com/sdboyer/cuetsy/encoder"
+)
+
+var generatedFileName = 0
+
+func (cmd Command) generateDashboardTypeScripts(c utils.CommandLine) error {
+	dest := c.String("dest")
+	if err := generateTypeScriptFromCUE(dest, paths, load.BaseDashboardFamily); err != nil {
+		return err
+	}
+
+	if err := generateTypeScriptFromCUE(dest, paths, load.DistDashboardFamily); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func generateTypeScriptFromCUE(dest string, p load.BaseLoadPaths, loader func(p load.BaseLoadPaths) (schema.VersionedCueSchema, error)) error {
+	dash, err := loader(p)
+	if err != nil {
+		return err
+	}
+
+	inst, _ := dash.CUE().Reference()
```

Comment:
this would not work, i can't get intance from the reference, need to find another way.

Reply:
Just commenting to have written down what we verbally discussed on Monday - it'd be preferable to refactor cuetsy to take a `cue.Value` instead of a `cue.Instance`